lte/gateway/c/session_manager/EnumToString.cpp

```diff
@@ -213,4 +215,22 @@ std::string event_trigger_to_str(EventTrigger event_trigger) {
   }
 }
 
+std::string request_origin_type_to_str(
```

Comment:
Is this a new change? I see it already on master: https://sourcegraph.com/github.com/magma/magma/-/blob/lte/gateway/c/session_manager/EnumToString.cpp